### DISCUSSION TITLE
"Ctrl+click" on xml parameter definition: long delay and error message

### DISCUSSION THREAD

**Discussion by WolfgangHG** [EWSoftware/SHFB#1024] [General]
Hi,

since a long time, I am annoyed that when CTRL+clicking on a xml comment param name (which happens to me if I want to replace it with a pasted text and want to select the full text), there is a really long Visual Studio hang (solution is large), and finally, a error message is shown:
![navigate_to_member_definition](https://github.com/EWSoftware/SHFB/assets/138049/3e435a4a-6a9c-44b2-9818-7848c220955f)

Here is a code snippet - I CTRL+clicked on the param comment for "_qualifikationsmodus":

```
    /// <summary>
    /// ...
    /// </summary>
    /// <param name="_dtVonBisZeitraum"></param>
    /// <param name="_colQualifikationenErforderlich"></param>
    /// <param name="_colQualifikationenAusschluss"></param>
    /// <param name="_qualifikationsmodus"></param>
    /// <returns></returns>
    public bool CheckQualifikationen(HGDateTimeVonBis _dtVonBisZeitraum, AdrQualifikationCollection _colQualifikationenErforderlich, AdrQualifikationCollection _colQualifikationenAusschluss, AufgabeQualifikationModus _qualifikationsmodus)
    {
```

Today, I found a sandcastle setting that disables this CTRL+click behavior:
![sandcastle_setting](https://github.com/EWSoftware/SHFB/assets/138049/2000b929-f523-419e-bb3e-31fd83d8aba0)

Though this is fine for me, I wonder what should actually happen here? Should it jump to the enum "AufgabeQualifikationModus"? If yes: this is found in another project in the current solution?
Any why is there this long delay (feels like a minute)? What happens under the hood?

Best regards

Wolfgang

**Comment by EWSoftware** [comment]
In earlier versions of Visual Studio, Go to Definition wasn't implemented for `name` and `cref` attribute values so the SHFB package implemented it.  It was disabled for `cref` attributes in VS 2017 and later but I missed the `name` attribute.  I've updated it to ignore it for those and it will let Visual Studio handle them starting with the next release.

**Comment by WolfgangHG** [comment]
Thanks, good to hear!
